roles/cfg_openwrt/templates/common/rc.local.j2

```diff
@@ -21,3 +21,12 @@ sed -i '/^ \/etc/d' /usr/lib/opkg/status
 {% for override in sysfs_overrides | default([]) %}
 echo {{ override['value'] }} > {{ override['path'] }}
 {% endfor%}
+
+{% if role == 'uplink_gateway' %}
```

Comment:
Cant you ship that cronjob with the package? Furthermore i find it more appropriate to cleanup more frequently that once a day. What about every 10min?